syntax = "proto3";

package Messages;

message EmptyAcknowledge
{
    
}